docs/field-details.asciidoc

```diff
@@ -4630,6 +4373,10 @@ example: `/home/alice`
 
 ==== Field Reuse
 
+The `process` fields are expected to be nested at: `process.parent`.
```

Comment:
Have you thought of ways to explain what each nesting means? I think we just copy and paste fieldsets over, but I don't know if we explain what the context of each nesting should mean.

Reply:
The new structure for how we specify field reuse will easily allow for that:

```YAML
  reusable:
    top_level: true
    expected:
      - at: process
        as: parent
        description: The process that gave birth to the one detailed at the root of the event.
```

And I think it's very much needed. However I think we should tackle this independently, as this will be a significant endeavour to add all of these contextual definitions and redesign the docs section about field reuse.